spdx/v2/v2_3/tagvalue/reader/parse_creation_info.go

```diff
@@ -94,6 +94,8 @@ func (parser *tvParser) parsePairFromCreationInfo(tag string, value string) erro
 		return parser.parsePairForAnnotation(tag, value)
 	case "AnnotationComment":
 		return parser.parsePairForAnnotation(tag, value)
+	case "DocumentComment":
+		return parser.parsePairForDocument(tag, value)
```

Comment:
```suggestion
		return parser.parsePairFromStart(tag, value)
```

And get rid of the `parsePairForDocument` function.

Reply:
Fixed, thanks!